components/cronet/native/url_request.cc

```diff
@@ -407,8 +407,6 @@ Cronet_RESULT Cronet_UrlRequestImpl::InitWithParams(
   for (const auto& request_header : params->request_headers) {
     if (request_header.name.empty())
       return engine_->CheckResult(Cronet_RESULT_NULL_POINTER_HEADER_NAME);
-    if (request_header.value.empty())
```

Comment:
is empty alias for empty string or is it for Null pointers as well? what I am trying to say, is there any risk?

Reply:
I have tested passing empty string ("") and not any value, which essentially is null value here in this check and it was able to catch both of that. 
But in our case, there is no risk as such. Because golang has no concept of null and strings are empty which should get converted to empty char array from cgo converter, so we're good. :)